src/components/form/form.types.ts

```diff
@@ -125,6 +125,12 @@ export interface LimeSchemaOptions {
      */
     collapsible?: boolean;
 
+    /**
+     * When `collapsible` is `true`, set this to `true` to make the
+     * collapsible section default to the open state
+     */
+    collapsibleDefaultOpen?: boolean;
```

Comment:
No, I guess that would work… Not sure why I wanted to tie it so specifically to `collapsible`…

Reply:
Oh… hmm… calling it `collapsed` means we'd have to default it to `true` if not set, otherwise we're changing the existing behaviour… I mean, we could do that of course, but it's a little less intuitive to have the default behaviour be that it loads collapsed, and the way to make it load open is to set `collapsed: false`… 🤔

Reply:
We could go with `expanded`? Or `loadExpanded` or something?

Reply:
I like short and concise names, so I vote for either `collapsed` or `expanded` :slightly_smiling_face: 
I don't think having `collapsed` default to `true` is _that_ weird, but `expanded` should work as well

Reply:
⚡